server/server.go

```diff
@@ -153,12 +153,14 @@ func (s *Server) restore() {
 	}
 	pods := map[string]*storage.RuntimeContainerMetadata{}
 	podContainers := map[string]*storage.RuntimeContainerMetadata{}
+	names := map[string][]string{}
```

Comment:
Nit but I'd use this instead:
```go
names := make(map[string][]string)
```

Reply:
how much is it important for you? :-)  I am fine either ways, but honestly I'd just avoid the CI circus if possible

Reply:
It's fine, that's just a nit :)